zingolib/src/lightclient/test_features.rs

```diff
@@ -4,22 +4,53 @@
 //! raw_receiver.0:   A &str representing the receiver address
 //! raw_receiver.1:   A u64 representing the number of zats to be sent
 //! raw_receiver.2:   An Option<&str> that contains memo data if not None
```

Comment:
im not sure who wrote this and when it merged but this comment is wrong. a raw_receiver is only used in integration testing not from the command line. also maybe "stuple" is a spelling mistake?

Reply:
`stuple` decapitated